api/src/main/java/com/github/retrooper/packetevents/protocol/item/type/ItemTypes.java

```diff
@@ -104,68 +153,68 @@ public class ItemTypes {
     public static final ItemType CYAN_TERRACOTTA = builder("cyan_terracotta").setMaxAmount(64).setPlacedType(StateTypes.CYAN_TERRACOTTA).build();
     public static final ItemType PRISMARINE_STAIRS = builder("prismarine_stairs").setMaxAmount(64).setPlacedType(StateTypes.PRISMARINE_STAIRS).build();
-    public static final ItemType IRON_BOOTS = builder("iron_boots").setMaxAmount(1).setMaxDurability(195).build();
+    public static final ItemType IRON_BOOTS = builder("iron_boots").setMaxAmount(1).setMaxDurability(195).setComponent(DAMAGE, 0).build();
     public static final ItemType BROWN_CONCRETE_POWDER = builder("brown_concrete_powder").setMaxAmount(64).setPlacedType(StateTypes.BROWN_CONCRETE_POWDER).build();
     public static final ItemType END_STONE = builder("end_stone").setMaxAmount(64).setPlacedType(StateTypes.END_STONE).build();
     public static final ItemType GLISTERING_MELON_SLICE = builder("glistering_melon_slice").setMaxAmount(64).build();
     public static final ItemType NETHER_SPROUTS = builder("nether_sprouts").setMaxAmount(64).setPlacedType(StateTypes.NETHER_SPROUTS).build();
     public static final ItemType GREEN_CONCRETE = builder("green_concrete").setMaxAmount(64).setPlacedType(StateTypes.GREEN_CONCRETE).build();
-    public static final ItemType ACACIA_DOOR = builder("acacia_door").setMaxAmount(64).setAttributes(ItemAttribute.FUEL).setPlacedType(StateTypes.ACACIA_DOOR).build();
-    public static final ItemType GOLDEN_AXE = builder("golden_axe").setMaxAmount(1).setAttributes(ItemAttribute.AXE, ItemAttribute.GOLD_TIER).setMaxDurability(32).build();
+    public static final ItemType ACACIA_DOOR = builder("acacia_door").setMaxAmount(64).setPlacedType(StateTypes.ACACIA_DOOR).setAttributes(ItemAttribute.FUEL).build();
+    public static final ItemType GOLDEN_AXE = builder("golden_axe").setMaxAmount(1).setMaxDurability(32).setAttributes(ItemAttribute.GOLD_TIER, ItemAttribute.AXE).setComponent(DAMAGE, 0).setComponent(ATTRIBUTE_MODIFIERS, new ItemAttributeModifiers(Arrays.asList(new ItemAttributeModifiers.ModifierEntry(Attributes.GENERIC_ATTACK_DAMAGE, new ItemAttributeModifiers.Modifier(UUID.fromString("cb3f55d3-645c-4f38-a497-9c13a33db5cf"), "Tool modifier", 6.0d, AttributeOperation.ADDITION), ItemAttributeModifiers.EquipmentSlotGroup.MAINHAND), new ItemAttributeModifiers.ModifierEntry(Attributes.GENERIC_ATTACK_SPEED, new ItemAttributeModifiers.Modifier(UUID.fromString("fa233e1c-4180-4865-b01b-bcce9785aca3"), "Tool modifier", -3.0d, AttributeOperation.ADDITION), ItemAttributeModifiers.EquipmentSlotGroup.MAINHAND)), true)).setComponent(TOOL, new ItemTool(Arrays.asList(new ItemTool.Rule(new MappedEntitySet<>(ResourceLocation.minecraft("incorrect_for_gold_tool")), null, false), new ItemTool.Rule(new MappedEntitySet<>(ResourceLocation.minecraft("mineable/axe")), 12.0f, true)), 1.0f, 1)).build();
```

Comment:
Oh this stuff seems a bit cursed with how long it is, I think we should at least make the UUIDS a static constant as they seem to be referenced in the other axes too